widgetssdk/src/main/java/com/glia/widgets/chat/ChatView.java

```diff
@@ -156,6 +164,9 @@ public void onItemRangeInserted(int positionStart, int itemCount) {
         }
     };
 
+    private Handler mainHandler = null;
```

Comment:
those two needs proper naming

Reply:
I think this naming is ok because it represents the main thread?

Reply:
oki, let it be for now